bundles.json

```diff
@@ -18,13 +18,13 @@
     "platform/features/scrolling",
     "platform/features/events",
     "platform/forms",
+    "platform/persistence/elastic",
     "platform/persistence/queue",
     "platform/policy",
     "platform/entanglement",
     "platform/search",
 
     "example/imagery",
-    "example/persistence",
     "example/eventGenerator",
```

Comment:
@slhale note that in development you can run the server with `node app.js -x example/persistence -i platform/persistence/elastic` to make these same changes without editing files.